site/reference/cli/fn/render/README.md

```diff
@@ -50,6 +50,11 @@ PKG_PATH:
   can perform privileged operations on your system, so ensure that binaries
   referred in the pipeline are trusted and safe to execute.
 
+--allow-network:
+  Allow functions to access network during pipeline execution. If unspecified,
+  `false` will be the default. Note that this is applicable for container
+  based functions only.
```

Comment:
suggested wording to make this less verbose:

```
Allow container-based functions to access network during pipeline 
execution. Default: `false`.
```

Reply:
Good call. Done.